services/core-api/Dockerfile.ci

```diff
@@ -0,0 +1,29 @@
+FROM python:3.6
+
+WORKDIR /app
+
+# Update installation utility
+RUN apt-get update
+
+# Install the requirements
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+# Fix log path permissions
```

Comment:
These permissions allow for volume mount locally 

in the cluster the logs must go via stdout to log aggregator. Since we use the default logger. 

If that is not the case, we should pipe all our logs to stdout in dev in proper JSON parable format. 


https://github.com/bcgov/mds/blob/ba754e73cf603752718cc05b27d6b0d61bea0f5d/services/core-api/app/config.py#L16-L43


Good for now but I would like to revisit this later

Reply:
Definitely worth revisiting and pointing out. Looking at the logs in Openshift, it outputs the health and liveness checks as well as API calls or errors. Let's keep this as is for now, but I'll create a ticket to address this.

Reply:
Following up with created ticket: https://bcmines.atlassian.net/browse/MDS-4338